src/pages/index.tsx

```diff
@@ -21,16 +31,103 @@ import tClientNext from "../tClientNext";
 import PublicUser from '../shared/publicModels/PublicUser';
 import PublicGroup from '../shared/publicModels/PublicGroup';
 import { MdVideocam } from 'react-icons/md';
+import { HSeparator } from 'horizon-ui/components/separator/Separator';
+import { IYuanjianUser } from 'shared/user';
+import { toast } from "react-toastify";
 
 const AppIndex: NextPageWithLayout = () => {
   const user = useUserInfo();
+  if(!user.name){
+    return <Box paddingTop={'80px'}><SetName u = {user} /><Meetings /></Box>
```

Comment:
do you need this box for modal?

Reply:
The page will not re-render since the modal closed will not update any properties on the meeting page, so my solution was to force users that have not entered names to enter a page with the modal